docker/scripts/mininet/multi_switch_mininet.py

```diff
@@ -85,7 +85,8 @@ def main():
 
     def formatParams(s):
         for param in params:
-            s = s.replace('{'+param+'}', str(params[param]))
+            s = s.replace('$'+param, str(params[param]))
```

Comment:
Awesome, thanks for making this change, Theo!

One minor tweak, I think, and then this is ready to merge. You should probably use a regex for these substitutions, because what you actually want to match here is something more like `"\$" + param + "\W"` - in other words, you want to be sure that you only match the param if it's not part of a larger word. If you have a param called `foo` which is set to `bar`, then `$foo` should yield `bar`, but `$foobaz` should not yield `barbaz`. (And along the same lines, it should be ok to have a param called `foo` and another called `foobar`.)